docs/data/stashcache/install-cache.md

```diff
@@ -199,62 +224,16 @@ The cache server functions as a normal HTTP server and can interact with typical
 user@host $ curl -O http://cache_host:8000/user/dweitzel/public/blast/queries/query1
 ```
 
-### Test Cache server reports to HTCondor collector
+### Test cache server reporting to the central collector
 
-To verify the cache is reporting to the central collector, run the following command:
+To verify the cache is reporting to the central collector, run the following command from the cache server:
 
 ```console
 user@host $ condor_status -any -l -const "Name==\"xrootd@`hostname`\""
 ```
 
-Where `hostname` is the string returned by the hostname command. The output of the above command should provide an HTCondor ClassAd that details the status of your cache.
-
-Registering the Cache
----------------------
-To be part of the OSG StashCache Federation, your cache must be
-[registered with the OSG](/common/registration.md).  The service type is `XRootD cache server`.
-
-The resource may also specify which VOs it will serve data from.
-To do this, add an `AllowedVOs` list, with each line specifying a VO whose StashCache data the resource is willing to cache.
-
-There are special values you can use in `AllowedVOs`:
-
-- `ANY_PUBLIC` indicates that the cache is willing to serve public data from any VO.
-- `ANY` indicates that the cache is willing to serve data from any VO,
-  both public and non-public.
-  `ANY` implies `ANY_PUBLIC`.
-
-There are extra requirements for serving non-public data:
-
-- In addition to the cache allowing a VO in the `AllowedVOs` list,
-  that VO must also allow the cache in its `AllowedCaches` list.
-  See the page on [getting your VO's data into StashCache](/data/stashcache/vo-data).
-- There must be an authenticated XRootD instance on the cache server.
-- There must be a `DN` attribute in the resource
-  with the DN of the cert used by the authenticated XRootD instance.
-
-This is an example of a cache server that serves all public data:
-```yaml
-  MY_STASHCACHE_CACHE:
-    FQDN: my-cache.example.net
-    Service: XRootD cache server
-      Description: StashCache cache server
-    AllowedVOs:
-      - ANY_PUBLIC
-```
-
-This is an example of a cache server that only serves authenticated data from the OSG VO:
-```yaml
-  MY_AUTH_STASHCACHE_CACHE:
-    FQDN: my-auth-cache.example.net
-    Service: XRootD cache server
-      Description: StashCache cache server
-    AllowedVOs:
-      - OSG
-    DN: /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=Services/CN=my-auth-cache.example.net
-```
+The output of the above command should provide an HTCondor ClassAd that details the status of your cache.
```

Comment:
Link to the HTCondor manual page about ClassAds here and maybe we should provide a snippet of output?

Reply:
I can do the snippet; perhaps I should leave off the bit about it being a ClassAd...